### ISSUE TITLE
ShopperLogin - getAccessToken - 415 Unsupported Media Type

### ISSUE THREAD

**Issue by andrewgadziksonos** [SalesforceCommerceCloud/commerce-sdk#348]
Hello!

I'm trying to use the `getAccessToken` function from the ShopperLogin client, and I'm getting a 415 Unsupported Media Type response.

Looking at the source code this is where the issue is,

```ts
return core_1.StaticClient.post({
      client: this,
      rawResponse: rawResponse,
      path: "/organizations/{organizationId}/oauth2/token",
      pathParameters: pathParameters,
      queryParameters: queryParameters,
      retrySettings: (options || {}).retrySettings,
      headers: headers,
      body: (options || {}).body,
    });
```

`core_1.StaticClient.post` internally is setting the `body` property to `body: JSON.stringify(options.body),` which is wrong for this particular API.  The body should be `body: new URLSearchParams(options.body).toString()` and the `Content-Type` header set to `application/x-www-form-urlencoded`

**Comment by wjhsf** [comment]
Hi @andrewgadziksonos, thanks for reporting this issue! We will have a fix released in the next week or two. I'll let you know as soon as the release is done.

**Comment by wjhsf** [comment]
This issue should be fixed in v2.6.1 of the SDK.
